### DISCUSSION TITLE
edges Add custom Position function

### DISCUSSION THREAD

**Discussion by sh4wnyue** [bcakmakoglu/vue-flow#1062] [Ideas]
Add a position for the edge. This can make the topology look better, position itself better, or merge edges and nodes. This can make the topology look better, position itself better, or merge edges and nodes.
  
- For example, in the picture below：   
 
![image](https://github.com/bcakmakoglu/vue-flow/assets/90391309/6b3f1580-6777-40b5-8def-8cf65c3df18c)

**Comment by bcakmakoglu** [comment]
Tbh I have no idea what this post is supposed to mean, you need to explain it properly.
Whatever you drew on your screenshot is already possible so I'm not sure what you're requesting here.

**Reply by bcakmakoglu** [reply]
Maybe you're looking for `sourcePosition` and `targetPosition` which already exist.
If you need to customize it further: use a custom edge.

**Comment by sh4wnyue** [comment]
The program is already very powerfully.


I mean, each node can customize the connection between up, down, left, and other nodes, and I looked at the documentation and they were all horizontal and up and down, and I couldn't find a single example of both up and down and left and right.

I tried the edge Position attribute and it doesn't seem to be supported，
I think we can add a Position attribute to the edge.

**Reply by bcakmakoglu** [reply]
You can use whatever handle direction you want. The example for horizontal does not exclude using vertical handles.

Again: Edges already supporting using `sourcePosition` and `targetPosition` (which should be equal to the position of the handle that is source/target), there's no need for additional props to do this. It's already very much possible.

[You might want to check the unidirectional example in the repo.](https://github.com/bcakmakoglu/vue-flow/tree/master/examples/vite/src/Unidirectional)

**Reply by sh4wnyue** [reply]
ok, thanks, I'll try！
